src/partials-js/data_fetch.js

```diff
@@ -4,40 +4,51 @@ import axios from 'axios';
 const $citySearchField = document.querySelector('form input');
 const $citySearchBtn = document.querySelector('form button');
 
-export let citiesData = [];
+export let cityData = null;
 export let defaultCity = [];
```

Comment:
napisałeś że nie jest juz tablicą co dla mnie było by lepiej a tu nadal tablica, poza tym sądze że globalna zmienna z danymi miasta powinna być jedna i na początku programu dostaje domyślne wartosci a potem jest nadpisywana przez wybrane miasto dzięki czemu możemy odnosić się tylko do jednej zmiennej bez konieczności sprawdzania czy juz 2 zmienna nie jest pusta

Reply:
fakt, zapomniałem zmienić na tablice, a to żeby zrobić tylko jedną nadpisywaną zmienną to dobry pomysł.
Już jest poprawione